src/main/java/com/group24/decide/LIC.java

```diff
@@ -60,14 +65,26 @@ public static boolean Condition1(){
      */
     public static boolean Condition2(){
         return true;
-    }    /**
+    }
+
+    /**
      *
      * @return Return true if the condition is met, other
      * returns false.
      */
-    public static boolean Condition3(){
-        return true;
-    }    /**
+    public boolean Condition3(){
```

Comment:
Maybe insert a text here where it describes what this conditions does 😃 So there is no need to double check the labinstructions everytime!